### PR TITLE
Add KSP-RO's ContractConfigurator

### PR DESCRIPTION
**Pull request by NathanKell**
This is a specialized version of CC for RP-1 games. It conflicts regular CC.

### CODE REVIEW

**Comment by NathanKell** [comment]
lol, forgot to mark the release as non-pre.

**Comment by NathanKell** [comment]
And...I'm a dumbo. Of course RP-1 *currently* deps CC, so recommending it results in a fail. Don't want to change its netkan until this is approved, though, obviously. So temporarily removing the RP-1 rec from the remote netkan.

**Comment by NathanKell** [comment]
There we go. Sorry about the test fail spam.

File: NetKAN/ContractConfiguratorRO.netkan [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "spec_version": "v1.16",
+    "identifier": "ContractConfiguratorRO",
```
Comment (HebaruSan) [inline NetKAN/ContractConfiguratorRO.netkan:3]:
```suggestion
    "identifier": "ContractConfigurator-RO",
```

The other RO-specific forked mods have a dash before `-RO`, let's keep this consistent with those.

image

**Comment by HebaruSan** [comment]
There are a lot of files in the `log` folder, which I would normally expect to see in someone's active `GameData` folder rather than a freshly packed mod. Is that intentional?

<details>
  <summary>Click to expand spoiler</summary>

```
    - GameData/ContractConfigurator/log
    - GameData/ContractConfigurator/log/!NO_GROUP
    - GameData/ContractConfigurator/log/!NO_GROUP/AsteroidLandingCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/CrewedSuborbital.log
    - GameData/ContractConfigurator/log/!NO_GROUP/CrewRotationStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/EarlyNavSats.log
    - GameData/ContractConfigurator/log/!NO_GROUP/EarlyWeatherSats.log
    - GameData/ContractConfigurator/log/!NO_GROUP/ExtraCrewStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/FirstCrewedLunarOrbit.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_Docking.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_EVA.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_MarsFlybyCrewed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_MoonFlybyCrewed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_MoonLandingCrewed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_OrbitCrewed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_OrbitRecover.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_spaceStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/first_VenusFlybyCrewed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyAriel.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyCallisto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyCeres.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyCharon.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyDeimos.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyDione.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyEnceladus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyEuropa.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyGanymede.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyIapetus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyIo.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyJupiter.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyMercury.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyMimas.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyMiranda.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyNeptune.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyOberon.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyPhobos.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyPluto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyRhea.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybySaturn.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyTethys.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyTitan.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyTitania.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyTriton.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyUmbriel.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyUranus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/flybyVesta.log
    - GameData/ContractConfigurator/log/!NO_GROUP/GEOWeather.log
    - GameData/ContractConfigurator/log/!NO_GROUP/JupiterFlybyCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingAsteroid.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingCallisto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingCharon.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingDeimos.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingDione.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingEnceladus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingEuropa.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingGanymede.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingIapetus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingIo.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingMars.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingMercury.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingMimas.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingPhobos.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingPluto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingRhea.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingTethys.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingTitan.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingTriton.log
    - GameData/ContractConfigurator/log/!NO_GROUP/landingVenus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/LifeSupportStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MarsLandingCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MarsOrbitCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MarsSpaceStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MarsSurfaceOutpost.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MarsSurfaceOutpostLab.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MercuryFlybyCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MolniyaRepeatComSats.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MoonExtendedStayCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MoonSpaceStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MoonSurfaceOutpost.log
    - GameData/ContractConfigurator/log/!NO_GROUP/MoonSurfaceOutpostLab.log
    - GameData/ContractConfigurator/log/!NO_GROUP/NewCrewStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OperationalNavSats.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OrbitalFlight1.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OrbitalFlight2.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OrbitalFlight3.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OrbitalTestFlight.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OrbitalTestFlightAnimal.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitCeres.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitCeres_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitJupiter.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitJupiter_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitMercury.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitMercury_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitNeptune.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitNeptune_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitPluto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/OrbitRecovery.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitSaturn.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitSaturn_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitUranus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitUranus_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitVesta.log
    - GameData/ContractConfigurator/log/!NO_GROUP/orbitVesta_Rep.log
    - GameData/ContractConfigurator/log/!NO_GROUP/PhobosLandingCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/PowerModuleStation.log
    - GameData/ContractConfigurator/log/!NO_GROUP/probeJupiter.log
    - GameData/ContractConfigurator/log/!NO_GROUP/probeMars.log
    - GameData/ContractConfigurator/log/!NO_GROUP/probeNeptune.log
    - GameData/ContractConfigurator/log/!NO_GROUP/probeSaturn.log
    - GameData/ContractConfigurator/log/!NO_GROUP/probeTitan.log
    - GameData/ContractConfigurator/log/!NO_GROUP/probeUranus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/recordCrewedAlt.log
    - GameData/ContractConfigurator/log/!NO_GROUP/recordCrewedCount.log
    - GameData/ContractConfigurator/log/!NO_GROUP/recordCrewedDuration.log
    - GameData/ContractConfigurator/log/!NO_GROUP/recordCrewedSpeed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/recordUncrewedAlt.log
    - GameData/ContractConfigurator/log/!NO_GROUP/recordUncrewedSpeed.log
    - GameData/ContractConfigurator/log/!NO_GROUP/Rendezvous.log
    - GameData/ContractConfigurator/log/!NO_GROUP/RepeatMoonLandingCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverCallisto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverCharon.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverDione.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverEnceladus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverEuropa.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverGanymede.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverIapetus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverIo.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverMars.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverMercury.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverMimas.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverPluto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverRhea.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverTethys.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverTitan.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverTriton.log
    - GameData/ContractConfigurator/log/!NO_GROUP/roverVenus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesAsteroid.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesCallisto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesCharon.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesDione.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesEnceladus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesEuropa.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesGanymede.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesIapetus.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesIo.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesMars.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesMercury.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesMimas.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesPhobos.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesPluto.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesRhea.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesTethys.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesTitan.log
    - GameData/ContractConfigurator/log/!NO_GROUP/samplesTriton.log
    - GameData/ContractConfigurator/log/!NO_GROUP/SaturnFlybyCrew.log
    - GameData/ContractConfigurator/log/!NO_GROUP/SecondGenNavSats.log
    - GameData/ContractConfigurator/log/!NO_GROUP/SecondGenWeather.log
    - GameData/ContractConfigurator/log/!NO_GROUP/SoundingDifficult.log
    - GameData/ContractConfigurator/log/!NO_GROUP/SuborbitalTestFlight.log
    - GameData/ContractConfigurator/log/!NO_GROUP/SunSyncWeather.log
    - GameData/ContractConfigurator/log/!NO_GROUP/TundraRepeatComSats.log
    - GameData/ContractConfigurator/log/!NO_GROUP/VenusOrbitCrew.log
    - GameData/ContractConfigurator/log/RP0
    - GameData/ContractConfigurator/log/RP0/AsteroidLandingCrew.log
    - GameData/ContractConfigurator/log/RP0/AtmoSat.log
    - GameData/ContractConfigurator/log/RP0/BreakSoundBarrier.log
    - GameData/ContractConfigurator/log/RP0/ComTestSat.log
    - GameData/ContractConfigurator/log/RP0/CrewedFlightAtmo.log
    - GameData/ContractConfigurator/log/RP0/CrewedFlightLow.log
    - GameData/ContractConfigurator/log/RP0/CrewedReachSpace.log
    - GameData/ContractConfigurator/log/RP0/CrewedReachSpaceDifficult.log
    - GameData/ContractConfigurator/log/RP0/CrewedSuborbital.log
    - GameData/ContractConfigurator/log/RP0/CrewedSupersonic.log
    - GameData/ContractConfigurator/log/RP0/CrewRotationStation.log
    - GameData/ContractConfigurator/log/RP0/DistanceSoundingDifficult.log
    - GameData/ContractConfigurator/log/RP0/Downrange.log
    - GameData/ContractConfigurator/log/RP0/DownrangeDifficult.log
    - GameData/ContractConfigurator/log/RP0/DownrangeIntermediate.log
    - GameData/ContractConfigurator/log/RP0/DownrangeSoundingIntermediate.log
    - GameData/ContractConfigurator/log/RP0/EarlyComNetwork3.log
    - GameData/ContractConfigurator/log/RP0/EarlyComNetwork4.log
    - GameData/ContractConfigurator/log/RP0/EarlyComSat.log
    - GameData/ContractConfigurator/log/RP0/EarlyNavSats.log
    - GameData/ContractConfigurator/log/RP0/EarlyWeatherSats.log
    - GameData/ContractConfigurator/log/RP0/ExtraCrewStation.log
    - GameData/ContractConfigurator/log/RP0/FirstComSat.log
    - GameData/ContractConfigurator/log/RP0/FirstCrewedLunarOrbit.log
    - GameData/ContractConfigurator/log/RP0/FirstGEOSat.log
    - GameData/ContractConfigurator/log/RP0/FirstMolniyaSat.log
    - GameData/ContractConfigurator/log/RP0/FirstNavSat.log
    - GameData/ContractConfigurator/log/RP0/FirstPolarSat.log
    - GameData/ContractConfigurator/log/RP0/FirstSatellite.log
    - GameData/ContractConfigurator/log/RP0/FirstScienceSat.log
    - GameData/ContractConfigurator/log/RP0/FirstSolarSat.log
    - GameData/ContractConfigurator/log/RP0/FirstSunSyncSat.log
    - GameData/ContractConfigurator/log/RP0/FirstTundraSat.log
    - GameData/ContractConfigurator/log/RP0/FirstWeatherSat.log
    - GameData/ContractConfigurator/log/RP0/first_Docking.log
    - GameData/ContractConfigurator/log/RP0/first_Downrange.log
    - GameData/ContractConfigurator/log/RP0/first_EVA.log
    - GameData/ContractConfigurator/log/RP0/first_FirstFlight.log
    - GameData/ContractConfigurator/log/RP0/first_GEOUncrewed.log
    - GameData/ContractConfigurator/log/RP0/first_KarmanCrewed.log
    - GameData/ContractConfigurator/log/RP0/first_KarmanUncrewed.log
    - GameData/ContractConfigurator/log/RP0/first_MarsFlybyCrewed.log
    - GameData/ContractConfigurator/log/RP0/first_MoonFlybyCrewed.log
    - GameData/ContractConfigurator/log/RP0/first_MoonFlybyUncrewed.log
    - GameData/ContractConfigurator/log/RP0/first_MoonImpact.log
    - GameData/ContractConfigurator/log/RP0/first_MoonLandingCrewed.log
    - GameData/ContractConfigurator/log/RP0/first_MoonOrbitUncrewed.log
    - GameData/ContractConfigurator/log/RP0/first_OrbitCrewed.log
    - GameData/ContractConfigurator/log/RP0/first_OrbitRecover.log
    - GameData/ContractConfigurator/log/RP0/first_OrbitScience.log
    - GameData/ContractConfigurator/log/RP0/first_OrbitUncrewed.log
    - GameData/ContractConfigurator/log/RP0/first_spaceStation.log
    - GameData/ContractConfigurator/log/RP0/first_VenusFlybyCrewed.log
    - GameData/ContractConfigurator/log/RP0/flybyAriel.log
    - GameData/ContractConfigurator/log/RP0/flybyCallisto.log
    - GameData/ContractConfigurator/log/RP0/flybyCeres.log
    - GameData/ContractConfigurator/log/RP0/flybyCharon.log
    - GameData/ContractConfigurator/log/RP0/flybyDeimos.log
    - GameData/ContractConfigurator/log/RP0/flybyDione.log
    - GameData/ContractConfigurator/log/RP0/flybyEnceladus.log
    - GameData/ContractConfigurator/log/RP0/flybyEuropa.log
    - GameData/ContractConfigurator/log/RP0/flybyGanymede.log
    - GameData/ContractConfigurator/log/RP0/flybyIapetus.log
    - GameData/ContractConfigurator/log/RP0/flybyIo.log
    - GameData/ContractConfigurator/log/RP0/flybyJupiter.log
    - GameData/ContractConfigurator/log/RP0/flybyMars.log
    - GameData/ContractConfigurator/log/RP0/flybyMercury.log
    - GameData/ContractConfigurator/log/RP0/flybyMimas.log
    - GameData/ContractConfigurator/log/RP0/flybyMiranda.log
    - GameData/ContractConfigurator/log/RP0/flybyNeptune.log
    - GameData/ContractConfigurator/log/RP0/flybyOberon.log
    - GameData/ContractConfigurator/log/RP0/flybyPhobos.log
    - GameData/ContractConfigurator/log/RP0/flybyPluto.log
    - GameData/ContractConfigurator/log/RP0/flybyRhea.log
    - GameData/ContractConfigurator/log/RP0/flybySaturn.log
    - GameData/ContractConfigurator/log/RP0/flybyTethys.log
    - GameData/ContractConfigurator/log/RP0/flybyTitan.log
    - GameData/ContractConfigurator/log/RP0/flybyTitania.log
    - GameData/ContractConfigurator/log/RP0/flybyTriton.log
    - GameData/ContractConfigurator/log/RP0/flybyUmbriel.log
    - GameData/ContractConfigurator/log/RP0/flybyUranus.log
    - GameData/ContractConfigurator/log/RP0/flybyVenus.log
    - GameData/ContractConfigurator/log/RP0/flybyVesta.log
    - GameData/ContractConfigurator/log/RP0/GeoComSatNetwork.log
    - GameData/ContractConfigurator/log/RP0/GEORepeatComSats.log
    - GameData/ContractConfigurator/log/RP0/GEOWeather.log
    - GameData/ContractConfigurator/log/RP0/HSFOrbitalLEO1Repeatable.log
    - GameData/ContractConfigurator/log/RP0/HSFOrbitalLEO2Repeatable.log
    - GameData/ContractConfigurator/log/RP0/HSFOrbitalLEO3Repeatable.log
    - GameData/ContractConfigurator/log/RP0/HSFOrbitalMoonGenRepeatable.log
    - GameData/ContractConfigurator/log/RP0/JupiterFlybyCrew.log
    - GameData/ContractConfigurator/log/RP0/KarmanLine.log
    - GameData/ContractConfigurator/log/RP0/landingAsteroid.log
    - GameData/ContractConfigurator/log/RP0/landingCallisto.log
    - GameData/ContractConfigurator/log/RP0/landingCharon.log
    - GameData/ContractConfigurator/log/RP0/landingDeimos.log
    - GameData/ContractConfigurator/log/RP0/landingDione.log
    - GameData/ContractConfigurator/log/RP0/landingEnceladus.log
    - GameData/ContractConfigurator/log/RP0/landingEuropa.log
    - GameData/ContractConfigurator/log/RP0/landingGanymede.log
    - GameData/ContractConfigurator/log/RP0/landingIapetus.log
    - GameData/ContractConfigurator/log/RP0/landingIo.log
    - GameData/ContractConfigurator/log/RP0/landingMars.log
    - GameData/ContractConfigurator/log/RP0/landingMercury.log
    - GameData/ContractConfigurator/log/RP0/landingMimas.log
    - GameData/ContractConfigurator/log/RP0/landingMoon.log
    - GameData/ContractConfigurator/log/RP0/landingMoonFarSide.log
    - GameData/ContractConfigurator/log/RP0/landingPhobos.log
    - GameData/ContractConfigurator/log/RP0/landingPluto.log
    - GameData/ContractConfigurator/log/RP0/landingRhea.log
    - GameData/ContractConfigurator/log/RP0/landingTethys.log
    - GameData/ContractConfigurator/log/RP0/landingTitan.log
    - GameData/ContractConfigurator/log/RP0/landingTriton.log
    - GameData/ContractConfigurator/log/RP0/landingVenus.log
    - GameData/ContractConfigurator/log/RP0/LifeSupportStation.log
    - GameData/ContractConfigurator/log/RP0/LunarFirstOrbiters.log
    - GameData/ContractConfigurator/log/RP0/LunarFlyby.log
    - GameData/ContractConfigurator/log/RP0/LunarImpactor.log
    - GameData/ContractConfigurator/log/RP0/MarsLandingCrew.log
    - GameData/ContractConfigurator/log/RP0/MarsOrbitCrew.log
    - GameData/ContractConfigurator/log/RP0/MarsOrbitRepeatable.log
    - GameData/ContractConfigurator/log/RP0/MarsSpaceStation.log
    - GameData/ContractConfigurator/log/RP0/MarsSurfaceOutpost.log
    - GameData/ContractConfigurator/log/RP0/MarsSurfaceOutpostLab.log
    - GameData/ContractConfigurator/log/RP0/MercuryFlybyCrew.log
    - GameData/ContractConfigurator/log/RP0/MolniyaRepeatComSats.log
    - GameData/ContractConfigurator/log/RP0/MoonExtendedStayCrew.log
    - GameData/ContractConfigurator/log/RP0/MoonLanding.log
    - GameData/ContractConfigurator/log/RP0/MoonLandingFarSide.log
    - GameData/ContractConfigurator/log/RP0/MoonLandingReturn.log
    - GameData/ContractConfigurator/log/RP0/MoonOrbiter.log
    - GameData/ContractConfigurator/log/RP0/MoonRover.log
    - GameData/ContractConfigurator/log/RP0/MoonSpaceStation.log
    - GameData/ContractConfigurator/log/RP0/MoonSurfaceOutpost.log
    - GameData/ContractConfigurator/log/RP0/MoonSurfaceOutpostLab.log
    - GameData/ContractConfigurator/log/RP0/NewCrewStation.log
    - GameData/ContractConfigurator/log/RP0/OperationalNavSats.log
    - GameData/ContractConfigurator/log/RP0/OrbitalFlight1.log
    - GameData/ContractConfigurator/log/RP0/OrbitalFlight2.log
    - GameData/ContractConfigurator/log/RP0/OrbitalFlight3.log
    - GameData/ContractConfigurator/log/RP0/OrbitalTestFlight.log
    - GameData/ContractConfigurator/log/RP0/OrbitalTestFlightAnimal.log
    - GameData/ContractConfigurator/log/RP0/orbitCeres.log
    - GameData/ContractConfigurator/log/RP0/orbitCeres_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitJupiter.log
    - GameData/ContractConfigurator/log/RP0/orbitJupiter_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitMars.log
    - GameData/ContractConfigurator/log/RP0/orbitMars_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitMercury.log
    - GameData/ContractConfigurator/log/RP0/orbitMercury_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitNeptune.log
    - GameData/ContractConfigurator/log/RP0/orbitNeptune_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitPluto.log
    - GameData/ContractConfigurator/log/RP0/OrbitRecovery.log
    - GameData/ContractConfigurator/log/RP0/orbitSaturn.log
    - GameData/ContractConfigurator/log/RP0/orbitSaturn_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitUranus.log
    - GameData/ContractConfigurator/log/RP0/orbitUranus_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitVenus.log
    - GameData/ContractConfigurator/log/RP0/orbitVenus_Rep.log
    - GameData/ContractConfigurator/log/RP0/orbitVesta.log
    - GameData/ContractConfigurator/log/RP0/orbitVesta_Rep.log
    - GameData/ContractConfigurator/log/RP0/PhobosLandingCrew.log
    - GameData/ContractConfigurator/log/RP0/PowerModuleStation.log
    - GameData/ContractConfigurator/log/RP0/probeJupiter.log
    - GameData/ContractConfigurator/log/RP0/probeMars.log
    - GameData/ContractConfigurator/log/RP0/probeNeptune.log
    - GameData/ContractConfigurator/log/RP0/probePluto.log
    - GameData/ContractConfigurator/log/RP0/probeSaturn.log
    - GameData/ContractConfigurator/log/RP0/probeTitan.log
    - GameData/ContractConfigurator/log/RP0/probeUranus.log
    - GameData/ContractConfigurator/log/RP0/probeVenus.log
    - GameData/ContractConfigurator/log/RP0/recordCrewedAlt.log
    - GameData/ContractConfigurator/log/RP0/recordCrewedCount.log
    - GameData/ContractConfigurator/log/RP0/recordCrewedDuration.log
    - GameData/ContractConfigurator/log/RP0/recordCrewedSpeed.log
    - GameData/ContractConfigurator/log/RP0/recordUncrewedAlt.log
    - GameData/ContractConfigurator/log/RP0/recordUncrewedSpeed.log
    - GameData/ContractConfigurator/log/RP0/Rendezvous.log
    - GameData/ContractConfigurator/log/RP0/RepeatMoonLandingCrew.log
    - GameData/ContractConfigurator/log/RP0/RocketPlaneDevelopment.log
    - GameData/ContractConfigurator/log/RP0/roverCallisto.log
    - GameData/ContractConfigurator/log/RP0/roverCharon.log
    - GameData/ContractConfigurator/log/RP0/roverDione.log
    - GameData/ContractConfigurator/log/RP0/roverEnceladus.log
    - GameData/ContractConfigurator/log/RP0/roverEuropa.log
    - GameData/ContractConfigurator/log/RP0/roverGanymede.log
    - GameData/ContractConfigurator/log/RP0/roverIapetus.log
    - GameData/ContractConfigurator/log/RP0/roverIo.log
    - GameData/ContractConfigurator/log/RP0/roverMars.log
    - GameData/ContractConfigurator/log/RP0/roverMercury.log
    - GameData/ContractConfigurator/log/RP0/roverMimas.log
    - GameData/ContractConfigurator/log/RP0/roverPluto.log
    - GameData/ContractConfigurator/log/RP0/roverRhea.log
    - GameData/ContractConfigurator/log/RP0/roverTethys.log
    - GameData/ContractConfigurator/log/RP0/roverTitan.log
    - GameData/ContractConfigurator/log/RP0/roverTriton.log
    - GameData/ContractConfigurator/log/RP0/roverVenus.log
    - GameData/ContractConfigurator/log/RP0/samplesAsteroid.log
    - GameData/ContractConfigurator/log/RP0/samplesCallisto.log
    - GameData/ContractConfigurator/log/RP0/samplesCharon.log
    - GameData/ContractConfigurator/log/RP0/samplesDione.log
    - GameData/ContractConfigurator/log/RP0/samplesEnceladus.log
    - GameData/ContractConfigurator/log/RP0/samplesEuropa.log
    - GameData/ContractConfigurator/log/RP0/samplesGanymede.log
    - GameData/ContractConfigurator/log/RP0/samplesIapetus.log
    - GameData/ContractConfigurator/log/RP0/samplesIo.log
    - GameData/ContractConfigurator/log/RP0/samplesMars.log
    - GameData/ContractConfigurator/log/RP0/samplesMercury.log
    - GameData/ContractConfigurator/log/RP0/samplesMimas.log
    - GameData/ContractConfigurator/log/RP0/samplesPhobos.log
    - GameData/ContractConfigurator/log/RP0/samplesPluto.log
    - GameData/ContractConfigurator/log/RP0/samplesRhea.log
    - GameData/ContractConfigurator/log/RP0/samplesTethys.log
    - GameData/ContractConfigurator/log/RP0/samplesTitan.log
    - GameData/ContractConfigurator/log/RP0/samplesTriton.log
    - GameData/ContractConfigurator/log/RP0/SaturnFlybyCrew.log
    - GameData/ContractConfigurator/log/RP0/SecondGenNavSats.log
    - GameData/ContractConfigurator/log/RP0/SecondGenWeather.log
    - GameData/ContractConfigurator/log/RP0/second_OrbitRecover.log
    - GameData/ContractConfigurator/log/RP0/SingleCrewedLunarOrbitRepeatable.log
    - GameData/ContractConfigurator/log/RP0/SoundingDifficult.log
    - GameData/ContractConfigurator/log/RP0/SoundingIntermediate.log
    - GameData/ContractConfigurator/log/RP0/SoundingRocketAdvancedBio.log
    - GameData/ContractConfigurator/log/RP0/SoundingRocketBio.log
    - GameData/ContractConfigurator/log/RP0/SoundingRocketDifficult.log
    - GameData/ContractConfigurator/log/RP0/SoundingRocketFilm.log
    - GameData/ContractConfigurator/log/RP0/SoundingRocketIntermediate.log
    - GameData/ContractConfigurator/log/RP0/SoundingRocketLow.log
    - GameData/ContractConfigurator/log/RP0/SuborbitalReturn.log
    - GameData/ContractConfigurator/log/RP0/SuborbitalTestFlight.log
    - GameData/ContractConfigurator/log/RP0/SunSyncWeather.log
    - GameData/ContractConfigurator/log/RP0/TundraRepeatComSats.log
    - GameData/ContractConfigurator/log/RP0/VenusOrbitCrew.log
```
</details>

File: NetKAN/ContractConfiguratorRO.netkan [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "$kref" : "#/ckan/netkan/https://raw.githubusercontent.com/KSP-RO/ContractConfigurator/master/ContractConfiguratorRO.netkan",
```
Comment (HebaruSan) [inline NetKAN/ContractConfiguratorRO.netkan:4]: Would you mind adding a ModuleManager depends to shut up our validator/bot?

image

image

**Comment by NathanKell** [comment]
Oops, meant to resolve-with-comment that I added MM.

Thanks for the catch re: logs! I nuked other in-use bits but missed those. Reupping bins.

**Comment by NathanKell** [comment]
I have a question about making use of this going forward, also. Right now, RP-1 deps CC. I would like to switch the dep to CC-RO, but since CC-RO conflicts with CC, as I understand it that means that users will (a) get the prompt to reinstall RP-1 because of changed metadata, but (b) that will fail because the reinstallation will see the user already has CC, which conflicts? Is there a way out of that?

**Comment by NathanKell** [comment]
Uh, I see it's tagged "waiting on PR creator" -- what else is needed?

**Comment by HebaruSan** [comment]
That was for the stuff you took care of, label has been removed.

I need to ponder the migration strategy question and also the existing contract packs that depend on `ContractConfigurator`. Would RP-1 users want to install these and use them? As this currently stands they would not be able to.

image

**Comment by NathanKell** [comment]
Nope, they would not--they're not compatible with RP-1, which is why this version of CC is explicitly marked as for-RP-1-only since they're also incompatible with *it*. This `conflicts` and explicitly does not `provides`.

**Comment by HebaruSan** [comment]
> I have a question about making use of this going forward, also. Right now, RP-1 deps CC. I would like to switch the dep to CC-RO, but since CC-RO conflicts with CC, as I understand it that means that users will (a) get the prompt to reinstall RP-1 because of changed metadata, but (b) that will fail because the reinstallation will see the user already has CC, which conflicts? Is there a way out of that?

I think you're right that it will work that way, and the only workaround I can think of so far is to remove the conflict temporarily and ask users to remove the old `ContractConfigurator` themselves, but the reinstall would still fail due to file collisions.

Sorry, that's all I've got; I thought of a few others ideas but they weren't any better. It seems like this is just inherently a tricky transition.

**Comment by NathanKell** [comment]
NP, that's what I figured, CKAN doesn't really have any support for migration.

Yeah, that means it needs to use a different path, then, so both can be installed at once (and a dll to yell at players to uninstall the old one).

Cheers!

**Pull request merged**